supplements/fizbans-treasury-of-dragons/items-magic.xml

```diff
@@ -0,0 +1,789 @@
+			<p class="indent">When you finish a long rest, choose a spell from any class list. The spell you choose must be of a level you can cast. You know the chosen spell and can cast it with your spell slots of the appropriate level until the end of your next long rest.</p>
+		</description>
+		<setters>
+			<set name="keywords">spellcasting, focus</set>
+			<set name="category">Wondrous Items</set>
+			<set name="cost" currency="gp">0</set>
+			<set name="weight" lb="0"></set>
+			<set name="slot">onehand</set>
+			<set name="type">Wondrous Item</set>
+			<set name="attunement" addition="by a Spellcaster">true</set>
+			<set name="rarity">Legendary</set>
+		</setters>
+	</element>
+
+	<element name="Sapphire Buckler" type="Magic Item" source="Fizban's Treasury of Dragons" id="ID_WOTC_FTOD_MAGIC_ITEM_SAPPHIRE_BUCKLER">
+		<description>
+			<p>This crystalline blue shield is fashioned from a sapphire dragon’s scale and is created to aid in rooting out the influence of Aberrations. While wielding the shield, you have resistance to psychic and thunder damage. Also, when you take damage from a creature that is within 5 feet of you, you can use your reaction to deal 2d6 thunder damage to that creature.</p>
+			<p class="indent">As an action, you can use the shield to help you locate Aberrations until the end of your next turn. If any Aberrations are within 1 mile of you, the shield emits a low humming tone for a moment, and you know the direction of all Aberrations within that range. Once this property is used, it can’t be used again until the next dawn.</p>
+		</description>
+		<setters>
+			<set name="keywords">thunder, aberrations</set>
+			<set name="category">Magic Armor</set>
+			<set name="cost" currency="gp">0</set>
+			<set name="type" addition="Shield">Armor</set>
+			<set name="rarity">Very Rare</set>
+			<set name="armor">ID_INTERNAL_ARMOR_GROUP_SHIELD</set>
+		</setters>
+		<rules>
+			<grant type="Condition" id="ID_INTERNAL_CONDITION_DAMAGE_RESISTANCE_THUNDER" />
+			<grant type="Condition" id="ID_INTERNAL_CONDITION_DAMAGE_RESISTANCE_PSYCHIC" />
+		</rules>
+	</element>
+
+	<element name="Topaz Annihilator" type="Magic Item" source="Fizban's Treasury of Dragons" id="ID_WOTC_FTOD_MAGIC_ITEM_TOPAZ_ANNIHILATOR">
+		<description>
+			<p>This magic ranged weapon resembles a musket, but in lieu of any ammunition, it holds a glowing yellow scale from a topaz dragon in its heart.</p>
+			<p class="indent">The weapon has a normal range of 100 feet and a long range of 300 feet, and it has the two-handed property. It deals 2d6 necrotic damage on a hit. If this damage reduces a creature or object to 0 hit points, the target is reduced to dust. A creature reduced to dust can be restored to life only by a true resurrection or wish spell.</p>
+			<p class="indent">While the weapon is on your person, you can use an action to cast the disintegrate spell (save DC 18). Once this property is used, it can’t be used again until the next dawn.</p>
```

Comment:
should be `<i>disintegrate</i>`

Reply:
```suggestion
			<p class="indent">While the weapon is on your person, you can use an action to cast the <i>disintegrate</i> spell (save DC 18). Once this property is used, it can’t be used again until the next dawn.</p>
```